transformation/src/main/java/org/opensearch/migrations/VersionMatchers.java

```diff
@@ -9,11 +9,15 @@ public class VersionMatchers {
     public static final Predicate<Version> isES_2_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 2.4"));
     public static final Predicate<Version> isES_5_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 5.6"));
     public static final Predicate<Version> isES_6_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 6.8"));
+    public static final Predicate<Version> isES_6_0_to_6_5 = VersionMatchers.equalOrGreaterThanMinorVersion(Version.fromString("ES 6.0"))
+                                                                .and(VersionMatchers.equalOrLessThanMinorVersion(Version.fromString("ES 6.5")));
+    public static final Predicate<Version> isES_6_5_orAbove = VersionMatchers.equalOrGreaterThanMinorVersion(Version.fromString("ES 6.5"));
     public static final Predicate<Version> isES_7_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 7.10"));
     public static final Predicate<Version> isES_7_10 = VersionMatchers.matchesMinorVersion(Version.fromString("ES 7.10.2"));
     public static final Predicate<Version> isES_8_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 8.17"));
-    public static final Predicate<Version> isES_7_0_to_7_9 = VersionMatchers.equalOrGreaterThanMinorVersion(Version.fromString("ES 7.0"))
-                                                                .and(VersionMatchers.equalOrLessThanMinorVersion(Version.fromString("ES 7.9")));
+    public static final Predicate<Version> isES_7_0_to_7_8 = VersionMatchers.equalOrGreaterThanMinorVersion(Version.fromString("ES 7.0"))
+                                                                .and(VersionMatchers.equalOrLessThanMinorVersion(Version.fromString("ES 7.8")));
+    public static final Predicate<Version> isES_7_9_X = VersionMatchers.matchesMinorVersion(Version.fromString("ES 7.9.0"));
     public static final Predicate<Version> equalOrGreaterThanES_7_10 = VersionMatchers.equalOrGreaterThanMinorVersion(Version.fromString("ES 7.10"));
```

Comment:
Do we need the => ES 7.10 independant of 7.9?  Seems like we can rename this existing matcher, no?